tests/x11/x3270_ssl.pm

```diff
@@ -42,6 +42,13 @@ sub run {
     # Install x3270
     zypper_call "install x3270";
 
+    # SLE-16633: [post GA] Update x3270 package for SLES 15 SP1 to support noverifycert option
+    # Check x3270 version
+    # x3270 3.6ga version has been submitted to SLE15 SP1, SP2, and SP3
+    zypper_call('info x3270');
+    my $current_ver = script_output("rpm -q --qf '%{version}' x3270");
```

Comment:
localhost:~ # rpm -q --qf %{version} kernel-default
5.3.185.3.18localhost:~ # 

Ben, seems the command needs some enhancement? why the output not in a single line?

Reply:
Good comment. I miss changing a new line. But it looks like no harm for catching the version. I will update it. Thanks.

Reply:
Modified done.